README.md

```diff
@@ -53,3 +53,5 @@ Table
 | @ElliotJKing                 | My favorite board game is Monopoly.                                          |
 | @scitics-io                  | I'm just testing
 | @fsansegundo                 | I have seen the Ibisbill                                        |
+| @chensmgt                    | Nice to try to contribute here.
+| @chensmgt                    | Figure out a new way to contribute.
```

Comment:
```suggestion
| @chensmgt                    | Nice to try to contribute here. |
| @chensmgt                    | Figure out a new way to contribute. |
```

I noticed we were missing the final `|` at the end of your row to close off the table. I added them for you :)